docs/scenarios/parametric.md

```diff
@@ -97,149 +92,50 @@ TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -k test_metrics_
-
-```json
-  "dns-opts": [
-    "single-request"
-  ],
+```sh
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -v
 ```
 
-
-#### Python
-
-To run the Python tests "locally" push your code to a branch and then specify ``PYTHON_DDTRACE_PACKAGE``.
-
+- `-vv`: Even more verbose output. Provides detailed information including setup and teardown for each test.
 
 ```sh
-TEST_LIBRARY=python PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-trace-py@2.x ./run.sh PARAMETRIC [-k ...]
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -vv -k test_metrics_
 ```
 
-#### NodeJS
-
-There is three ways for running the NodeJS tests with a custom tracer:
-1. Create a file `nodejs-load-from-npm` in `binaries/`, the content will be installed by `npm install`. Content example:
-    * `DataDog/dd-trace-js#master`
-2. Clone the dd-trace-js repo inside `binaries`
-3. Create a file `nodejs-load-from-local` in `binaries/`, this will disable installing with `npm install dd-trace` and
-   will instead get the content of the file, and use it as a location of the `dd-trace-js` repo and then mount it as a
-   volume and `npm link` to it. For instance, if this repo is at the location, you can set the content of this file to
-   `../dd-trace-js`. This also removes the need to rebuild the weblog image since the code is mounted at runtime.
-
-#### Ruby
+- `-s`: Disable output capture. Allows you to see print statements and logs directly in the console.
 
-There is two ways for running the Ruby tests with a custom tracer:
-
-1. Create an file ruby-load-from-bundle-add in binaries/, the content will be installed by bundle add. Content example:
-gem 'ddtrace', git: "https://github.com/Datadog/dd-trace-rb", branch: "master", require: 'ddtrace/auto_instrument'
-In addition, instead of `branch:`, `ref` can be used to reference a commit hash.
-We'd recommend this if you're going to be iterating as it causes the build
-script to always rebuild the ddtrace gem.
-2. Clone the dd-trace-rb repo inside binaries
-
-#### C++
-
-There is two ways for running the C++ library tests with a custom tracer:
-1. Create a file `cpp-load-from-git` in `binaries/`. Content examples:
-    * `https://github.com/DataDog/dd-trace-cpp@main`
-    * `https://github.com/DataDog/dd-trace-cpp@<COMMIT HASH>`
-2. Clone the dd-trace-cpp repo inside `binaries`
-
-The parametric shared tests can be run against the C++ library,
-[dd-trace-cpp][1], this way:
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC
+```sh
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -s
 ```
 
-Use the `-k` command line argument, which is forwarded to [pytest][2], to
-specify a substring within a particular test file, class, or method. Then only
-matching tests will run, e.g.
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC -k test_headers
-```
+### Running the tests for a custom tracer
+To run tests against custom tracers, refer to the [Binaries Documentation](system-tests/docs/execute/binaries.md)
```

Comment:
I think this link path may not work? Locally for me it jumps to docs/scenarios/system-tests/docs/execute/binaries.md which isn't a real file. I guess you need to use relative path instead, so `../execute/binaries.md' I think?